tests/application-gateway-tests/test/executor/proxy/suite.go

```diff
@@ -215,6 +215,10 @@ func (ts *TestSuite) AddDenierLabel(t *testing.T, apiId string) {
 
 	pod.Labels[serviceName] = "true"
 
-	_, err = ts.podClient.Update(pod)
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
I am not sure if this is correct usage of RetryOnConflict, please have a look here:
https://github.com/pingcap/tidb-operator/issues/81
Generally, we should fetch pod one more time, and again apply our changes.

Reply:
Good point! I do not think it impacts current implementation but we should definitely correct it in the future.